### PR TITLE
show other photographer if it is set

### PR DESCRIPTION
**Pull request by jamesgorrie**
We do not set `isOther` on load so if it was set, we just show and empty dropdown.
This makes sure we do.

other-photo

### CODE REVIEW

File: kahuna/public/js/usage-rights/usage-rights-editor.js [hunk 1]
```diff
@@ -134,6 +134,14 @@ usageRightsEditor.controller(
         return property.optionsMap[val];
     };
 
+    ctrl.isOther = property => {
```
Comment (akash1810) [inline kahuna/public/js/usage-rights/usage-rights-editor.js]: minor: would `isOtherOption` be more descriptive?

**Comment by akash1810** [comment]
:+1:

File: kahuna/public/js/usage-rights/usage-rights-editor.js (hunk 1 above)
Comment (jamesgorrie) [inline kahuna/public/js/usage-rights/usage-rights-editor.js]: I've gone with `isOtherValue`?

**Comment by theefer** [comment]
:+1:

File: kahuna/public/js/usage-rights/usage-rights-editor.js (hunk 1 above)
Comment (akash1810) [inline kahuna/public/js/usage-rights/usage-rights-editor.js]: :+1:

**Pull request merged**